locales/sv_SE/pcsx2_Iconized.po

```diff
@@ -282,6 +282,23 @@ msgid ""
 "set of games that have unusual performance requirements, but have adverse "
 "effects on most others. Not recommended for underpowered PCs."
 msgstr ""
+"Förinställningarna tillämpar UppsnabbningsTrimningar, några\n"
+" omkompilerarfunktioner, och en del Spelfixar som till vetskap ökar farten.\n"
+"\n"
+"FörinställningsInfo:\n"
+" 1  - Säkrast - Inga UppsnabbningsTrimningar.\n"
+"        Mest tillförlitligt, men möjligen också långsamt.\n"
+" 2  - Säkert - Förval. En del UppsnabbningsTrimningar\n"
+"         som till vetskap ökar farten, med minimala bieffekter.\n"
+" 3  - Balanserat - Kan hjälpa fyrkärniga (quadcore) CPU'er.\n"
+" 4  - Aggresivt - Kan hjälpa lågprestanda CPU'er för lågkrävande spel,\n"
+"         men kan vålla problem i andra sammanhang.\n"
+" 5  - Mycket Aggresivt - Kan hjälpa lågprestanda CPU'er\n"
+"         för lågkrävande spel, men lär vålla problem i andra sammanhang.\n"
+" 6  - Mest Skadligt - Hård tillämpning av UppsnabbningsTrimningar.\n"
+"         Kan hjälpa en mindre del spel med ovanliga prestandakrav,\n"
+"          men har en ogynnsam effekt för de flesta.\n"
+"        Avrådes för lågprestanda PC'er."
```

Comment:
This sentence is straight out weird, just rewrite it into something like 'Avrådes för PC:s med låg prestanda'.

Reply:
Weird perhaps, but I'm not in favor of changing "PC'er" into "PC:s".